### DISCUSSION TITLE
USA States Booster Vaccination Data

### DISCUSSION THREAD

**Discussion by visheshmadan** [owid/covid-19-data#2239] [Q&A] [answered]
Hi there,

Interested to know how to get USA booster doses data. US_state_vaccination.csv file doesn't appear to show information on boosters and I've been unable to find a public source for the information so far.
If I am trying to calculate Count of Booster dose= Total Vaccination-Single Vaccinated -Fully Vaccinated
Then it is not giving the correct number.

Thanks

**Comment by lucasrodes** [comment, marked as answer]
Hi @visheshmadan,
Thanks for bringing this to our attention. We have added two new metrics to [us_state_vaccinations.csv](https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/us_state_vaccinations.csv) file: `total_boosters` and `total_boosters_per_hundred`, as these were already available in our source ([COVID Data Tracker API by CDC](https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=vaccination_data)).

I hope this helps.

**Reply by visheshmadan** [reply]
Hi @lucasrodes 
Thanks for adding these columns on priority, it will help us in getting booster data at states level.
